org.gtimelog.GTimeLog.json

```diff
@@ -0,0 +1,44 @@
+{
+  "id": "org.gtimelog.GTimeLog",
+  "runtime": "org.gnome.Platform",
+  "sdk": "org.gnome.Sdk",
+  "runtime-version": "44",
+  "rename-icon": "gtimelog",
+  "rename-desktop-file": "gtimelog.desktop",
+  "rename-appdata-file": "gtimelog.appdata.xml",
+  "command": "gtimelog",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--share=network",
+    "--own-name=org.gtimelog",
+    "--own-name=org.mpris.MediaPlayer2.org.gtimelog.GTimeLog",
```

Comment:
I don't think this is necessary?

Reply:
you tell me. `--log-session-bus=` was showing it.

Line 15 however is unfortunate because no one should register/own that bus name.

Reply:
(it works without though)

Reply:
I cannot explain the mpris thing.  I'm definitely not doing anything involving mpris, it must've been something else in the stack.

I remember having doubts about using plain `org.gtimelog` instead of `org.gtimelog.GTimeLog` back when I first started porting to GTK 3.  Should I switch before it's too late?

i_have_no_idea_what_Im_doing.gif

Reply:
this is contra to the spec.

Reply:
Which spec?

Reply:
removed the mpris name. Not sure why it appear in the dbus logging, but it doesn't prevent the app to run.

Reply:
TIL. this is clearer now. the mpris bus is automatically granted by flatpak based on the app-id. Which is why it appeared in the bus-session-log.